src/pymedphys/_labs/paulking/profile.py

```diff
@@ -24,27 +24,56 @@
 # program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
 
 
+from copy import deepcopy
 from typing import Callable
 
 import numpy as np
 from scipy import interpolate
 import matplotlib.pyplot as plt
 
+import xarray as xr
+
 from ...libutils import get_imports
-from ...xarray import XArrayComposition
```

Comment:
Did you delete this import on purpose?

Reply:
No, not intentionally. 
But, if I add that that import back and edit as follows ...

class Dose1D(XArrayComposition):

... then it fails an assertion in one of your tests.

Reply:
All good, can sort that out later if you like.